tests/test_kubeseal.py

```
@@ -1,182 +1,283 @@
+            assert "--dry-run=client" in cmd
+            assert "-o" in cmd
+            assert "yaml" in cmd
+
+    def test_create_regcred_secret(self, kubeseal_mocks, mock_subprocess):  # noqa: ARG002
+        """Test creating a docker-registry secret."""
+        kubeseal = Kubeseal(select_context=False)
+        secret_params = {"name": "test-regcred-secret", "namespace": "default", "type": "docker-registry"}
+
+        docker_server = "https://index.docker.io/v1/"
+        docker_username = "testuser"
+        docker_password = "testpassword"
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "docker_password"

(S105)